compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/Mnemonics.java

```diff
@@ -135,6 +135,8 @@
         mnemonics[InstructionCodes.FLT] = "flt";
         mnemonics[InstructionCodes.ILE] = "ile";
         mnemonics[InstructionCodes.FLE] = "fle";
+
+        mnemonics[InstructionCodes.INT_RANGE] = "int_range";
```

Comment:
Need to update the **org.ballerinalang.util.codegen.Mnemonics** class as well.

Reply:
Already done at https://github.com/ballerina-platform/ballerina-lang/pull/8840/files#diff-ddd1b62b5ba6ed27cd24235a3a369dfbR139

Reply:
Seems like I somehow missed it. Sorry about the noise.

Reply:
Place this in the correct order.